src/configuration.cc

```diff
@@ -156,6 +162,25 @@ int Configuration::ParseArguments(int ac, char** av) {
                 << std::endl;
       return -1;
     }
+    if (flags.count("options")) {
+      std::string input_str;
+      const std::vector<std::string> options =
+          flags["options"].as<std::vector<std::string>>();
+      for (const std::string& option: options) {
+        std::vector<std::string> key_value;
+        boost::algorithm::split(key_value, option, boost::is_any_of("="));
+        if (key_value.size() != 2) {
+          std::cerr << "Invalid option " << option;
+          return 1;
+        }
+        input_str += key_value[0] + ": " + key_value[1] + "\n";
+      }
+      if (verbose_logging_) {
+        std::cout << "Options:\n" << input_str;
+      }
+      std::istringstream input { input_str };
```

Comment:
The convention elsewhere in the code is to use parentheses.

Reply:
outdated line